doc/internal/release.rst

```diff
@@ -15,28 +15,34 @@ This document is only meant for the project administrators, not users and develo
 
    d. run ``make linkcheck`` on the documentation and fix any redirected/broken links.
 
-   e. update the README.
+   e. update the README and this release documentation, if necessary.
 
-3. Build the new conda package locally on your mac using the following command  (*Note*: you may have to replace the contents of the ``requirements()`` function in ``setup.py`` with a ``pass`` statement to get ``conda build`` to work)::
+3. Build the new conda package using the following command::
 
-    conda build -c defaults -c conda-forge --python=3.6 --numpy=1.14 skll
+    conda build -c conda-forge --numpy=1.17 skll
 
-4. Convert the package for both linux and windows::
+4. Upload the package to anaconda.org using ``anaconda upload --user ets <package tarball>``. You will need to have the appropriate permissions for the ``ets`` organization. 
 
-    conda convert -p win-64 -p linux-64 <mac package tarball>
+5. Build the PyPI source distribution using ``python setup.py sdist build``.
 
-5. Upload each of the packages to anaconda.org using ``anaconda upload <package tarball>``.
+6. Upload the source distribution to TestPyPI  using ``twine upload --repository testpypi dist/*``. You will need to have the ``twine`` package installed and set up your ``$HOME/.pypirc`` correctly. See details `here <https://packaging.python.org/guides/using-testpypi/>`__.
 
-6. Upload source package to PyPI using ``python setup.py sdist upload``.
+7. Test the conda package by creating a new environment on different platforms with this package installed and then running SKLL examples or tests from a SKLL working copy. If the package works, then move on to the next step. If it doesn't, figure out why and rebuild and re-upload the package.
 
-7. Draft a release on GitHub.
+8. Test the TestPyPI package by installing it as follows: ``pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple skll`` and then running SKLL examples or tests from a SKLL working copy. If the package works, then move on to the next step. If it doesn't, figure out why and rebuild and re-upload the package.
```

Comment:
nitpick
`pip install` command is going out of the page, can we make it to multiple lines. i looked in chrome and safari and both have similar view. 
![Screen Shot 2019-10-24 at 9 44 42 AM](https://user-images.githubusercontent.com/3886249/67491528-15c53c00-f643-11e9-8d32-6fc1ebf941d4.png)